# Exceptions
We used exceptions, then handled them by using try and catch. We also used throw.
